docs/file-reference.md

```diff
@@ -0,0 +1,367 @@
+# OpenComposition file reference
+
+Each file defines one micro-service, which forms one `pod` controlled by it's
+controller(right now the default controller is `deployment`).
+
+
+A example using all the keys added in OpenComposition(not all keys from Kubernetes
+API are included):
+
+```yaml
```

Comment:
@kadel I agree that we can/should have a simpler example but since this is "reference" it should be complete with all possible things with this tool. So let's not remove the complete example. But we can add a nice and short example to start with. Explain it in short and then show the full power. WDYT?